business_requirement_deliverable_project_task_categ/__manifest__.py

```diff
@@ -9,6 +9,11 @@
     "website": "https://www.elico-corp.com/",
     "author": "Elico Corp, Odoo Community Association (OCA)",
     "license": "AGPL-3",
+    'external_dependencies': {
```

Comment:
you will need to bump version number too

Reply:
Does it need to modify version?    In business_requirement_deliverable_resource_template_categ/__manifest__.py, modify version to 10.0.1.1.0, because I modified model business.requirement.resource.template, update field categ_id link previous project.task to now project.category.  So I don't think should be modify version number if just add a migration script.  What about you? @elicoidal

Reply:
At least you need modify the lower number (restart needed).
Still not sure where the PR should go: whether in openupgrade or in this repo (https://github.com/OCA/project/pull/386#pullrequestreview-127355453)